tests/test_ext_commands.py

```diff
@@ -202,8 +206,28 @@ def test_sanitize_repo_return_fail_when_has_untracked_files(
         "--no-commit"
     )
 
-    assert result.status == plug.Status.ERROR
-    assert "untracked file" in result.msg
+    expected_error = result["sanitizer"][0]
+
+    assert expected_error.status == plug.Status.ERROR
+    assert "untracked file" in expected_error.msg
+
+
+def test_sanitize_repo_returns_success_status(sanitizer_config, fake_repo):
+    """If nothing goes wrong, this test should find that sanitize_repo
+    returns a plug.Status with a SUCCESS
+    """
+    target_branch = "student-version"
+
+    result = execute_sanitize_repo(
+        f"--file-list {fake_repo.file_list_path} "
+        f"--repo-root {fake_repo.path} "
+        f"--target-branch {target_branch} "
+    )
+
+    expected_success = result["sanitizer"][0]
```

Comment:
Even though this works, we probably want to check if any element inside `result["sanitizer"]´ has the error we're looking for

Reply:
Gone and fixed it

Reply:
Though now its possible that if the tests fail it is unclear what causes it

Reply:
It's fine